src/runtime/tailwind.config.ts

```diff
@@ -2,6 +2,7 @@ import defaultColors from 'tailwindcss/colors.js';
 // import {type Config} from "tailwindcss";
 
 const colors = {
+  'transparent': defaultColors.transparent,
```

Comment:
Please give me a hint on such thigs like adding a new color. I must keep it in sync with figma.